flask_project/campaign_manager/test/test_intergration.py

```diff
@@ -0,0 +1,185 @@
+import os
```

Comment:
there is a typo error in the file name: test_integration.py instead of test_intergration.py :)